.github/actions/external_runner/main.go

```diff
@@ -39,6 +42,10 @@ const (
 	logLinesReqBackoff = 5 * time.Second
 )
 
+var (
```

Comment:
WDYT if we move this to the action parameter? Then we can remove disable_logs and if no filter is applied - no logs will be shown.

Reply:
Good point! That way will be much flexible too, so we don't couple the Go code with the type of command the Fargate task is running (Ansible, etc)

Thank you

Reply:
Added in https://github.com/newrelic/infrastructure-agent/pull/1335/commits/22134483d906aa5bcecc3c5c1ca7e6c690cf4bb3